content/en/tutorial/01-vdom.md

```diff
@@ -151,6 +150,24 @@ export default function App() {
 		<p class="big">Hello World!</p>
 	)
 }
+
+render(<App />, document.getElementById("app"));
+```
+
+When passing a component to `render()`, it's important to let Preact do the
+instantiation rather than invoking your component directly, which will break
+in unexpected ways:
+
+```jsx
+const App = () => <div>foo</div>;
+
+// DON'T: Invoking components directly breaks hooks and update ordering:
+render(App(), rootElement); // ERROR
+render(App, rootElement); // ERROR
+
+// DO: Passing components using createElement() or JSX allows Preact to render correctly:
+render(createElement(App), rootElement); // success
+render(<App />, rootElement); // success
```

Comment:
Pretty much directly copied from the [API reference](https://preactjs.com/guide/v10/api-reference#render) but with a switch to `createElement` as we use it, rather than `h`, on this page.